voytovsky-alexey/hw3-2-translator.js

```diff
@@ -0,0 +1,32 @@
+const readline = require('readline');
+const request = require('request');
+
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout
+});
+
+const program = () =>
+
+	rl.question('Yandex EN-RU translator:\n\n', (data) => {
+		const apiKey = 'trnsl.1.1.20190915T202449Z.94d7a5f7880c0675.592e3cc378620ec06c12022fd747b838c01086c0';
+		const translator = `https://translate.yandex.net/api/v1.5/tr.json/translate?key=${apiKey}&lang=en-ru&text=${data}`;
+
+		request(translator, (err, response, body) => {
```

Comment:
Можно вот так еще, тогда не нужно будет парсить json ручками)
```suggestion
		request({url: translator, json: true}, (err, response, body) => {
```

Reply:
Круто, не знал! Учту!